libs/use-publisher/src/index.ts

```diff
@@ -47,13 +48,17 @@ const usePublisher = ({
 
       const { label, mediaStream } = stream;
 
+      const allSourceIds = Array.from(sources).map(([, { broadcastOptions }]) => broadcastOptions.sourceId);
+
+      const dedupedSourceId = validateDuplicateSourceIds(label ?? streamId, allSourceIds);
+
       const broadcastOptions = {
         bandwidth: bitrateList[0].value,
         codec: codecList[0],
         events: ['viewercount'] as Event[],
         mediaStream,
         simulcast: codecList[0] === 'h264',
```

Comment:
We can only enable simulcast when
1. in Chrome browser
2. codec is h264 or vp8